deflect/config.h

```diff
@@ -45,8 +45,10 @@
 #    define final
 #    define override
 #  endif
-#  ifndef CXX_NOEXCEPT_SUPPORTED
-#    define noexcept
+#  ifdef CXX_NOEXCEPT_SUPPORTED
+#    define NOEXCEPT noexcept
```

Comment:
That's not pretty.. I don't understand why, but I guess you had no choice on MSVC?